### DISCUSSION TITLE
Files not loading with: __STATIC_CONTENT is not defined

### DISCUSSION THREAD

**Discussion by dev-on2air** [flareact/flareact#194] [Q&A]
I'm getting following error: __STATIC_CONTENT is not defined

My worker code is running, but any other static files uses are not loading.  It appears to be coming from kv-asset-handler.

I've now created a KV Namespace called __STATIC_CONTENT and added to my wrangler.toml but still having same issue.

Something I'm doing wrong?  I didn't find any reference to having to configure a __STATIC_CONTENT KV.

**Comment by tj-kev** [comment]
Do you have an example repo I could take a look at?

**Reply by dev-on2air** [reply]
Sure - here is example codebase:
https://github.com/on2air/sample-flareact

And here is example site:  https://test-flareact.o2adev.com/page/067dd719a912471ea9a3ac10710e7fdf

You'll see that it is running the [slug].js file correctly, but the css/js files being loaded from /_flareact/static folder are all failing with '__STATIC_CONTENT is not defined'

I'm using this library primarily to generate a notion page: https://github.com/NotionX/react-notion-x

It works correctly when running locally via 'flareact dev'

**Reply by tj-kev** [reply]
You need to update your wrangler.toml file slightly.

bucket and entrypoint should be in their own section like below:

```toml
[site]
bucket = "out"
entry-point = "./"
```

Documentation for that bit available here: [https://developers.cloudflare.com/workers/platform/sites/configuration#wranglertoml](https://developers.cloudflare.com/workers/platform/sites/configuration#wranglertoml)

**Comment by dev-on2air** [comment]
thank you @tj-kev - I assumed that was for setting up variables under an environment.  That fixed initial issue.  Now it appears that I am unable to import a css file from an imported library:

`import 'react-notion-x/src/styles.css'`

This doesn't add any css to the output file.  Worst case I could add that file into the code of my library but would prefer to keep in its library.  Is that supported?

**Reply by tj-kev** [reply]
Not come across this issue before. Looks like its caused by the "sideEffects":false in the react-notion-x package.json. Changing it to true manually suddenly fixes the issue. But obviously that is not practical.

I found this issue on the webpack css plugin which mentions it, with a fix: [https://github.com/webpack-contrib/mini-css-extract-plugin/issues/102#issuecomment-384947142](https://github.com/webpack-contrib/mini-css-extract-plugin/issues/102#issuecomment-384947142)

You should be able to add this to a custom webpack config detailed here: [https://flareact.com/docs/custom-webpack-config](https://flareact.com/docs/custom-webpack-config).

Hope this helps.

**Reply by dev-on2air** [reply]
I'll take a look. appreciate the help!
